plugin/tagbar-markdown.vim

```diff
@@ -0,0 +1,57 @@
+
+if exists("tagbar_markdown")
+    finish
+endif
+let tagbar_markdown = 1
+
+
+if (!exists('g:markdown2ctags'))
+    let g:markdown2ctags = '~/.vim/bundle/markdown2ctags/markdown2ctags.py'
```

Comment:
I think there's probably a better way to do this without hard-coding the path quite like this.